### DISCUSSION TITLE
4.1.1 contains sources and everything. is that expected?

### DISCUSSION THREAD

**Discussion by KillerCodeMonkey** [the-guild-org/apollo-angular#1852] [General]
heyho,

instead of creating a bug, i just first wanna ask, if this build/release was wrongly published?

seems like you released the whole project and not only the build- folder.

Am i right?

**Comment by elliotleelewis** [comment]
I also have failing unit tests after updating to v4.1.1. See here:

https://github.com/elliotleelewis/home-system/pull/411

![image](https://user-images.githubusercontent.com/17044667/204605047-903b608f-caa8-4549-b45e-e49445b6c8c3.png)
